local-cli/link/ios/registerNativeModule.js

```diff
@@ -10,19 +10,16 @@
 const xcode = require('xcode');
 const fs = require('fs');
 const path = require('path');
-const log = require('npmlog');
 
 const addToHeaderSearchPaths = require('./addToHeaderSearchPaths');
 const getHeadersInFolder = require('./getHeadersInFolder');
 const getHeaderSearchPath = require('./getHeaderSearchPath');
-const getProducts = require('./getProducts');
```

Comment:
Looks like `getProducts.js` isn't accessed anywhere else and can be deleted too.

Reply:
I'll remove it.